### ISSUE TITLE
Add an option to use the system default font

### ISSUE THREAD

**Issue by mon-jai** [Mosc/Glider#94]
It would be nice if we can change app font to Roboto instead.

**Comment by Mosc** [comment]
Duplicate of #49. Unfortunately this is blocked by the Flutter framework.

**Comment by mon-jai** [comment]
@Mosc How about adding Roboto as a font option?

It serves as Android's default font and looks closer to the iOS default font compared to Noto Sans.

**Comment by Mosc** [comment]
That seems like a reasonable compromise. I will consider that.
